tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/executor/TestExecutor.java

```diff
@@ -303,5 +303,56 @@ public void test05() throws TornadoExecutionPlanException {
         }
     }
 
+    /**
+     * Test Multi-Graphs in an execution plan. An execution plan can hold and launch
+     * multiple immutable task graphs. This tests shows how to execute individual immutable
+     * task graphs in any order.
+     * 
+     * @throws TornadoExecutionPlanException
+     */
+    @Test
+    public void test06() throws TornadoExecutionPlanException {
+        int numElements = 16;
+        IntArray a = new IntArray(numElements);
+        IntArray b = new IntArray(numElements);
+        IntArray c = new IntArray(numElements);
+
+        a.init(1);
+        b.init(2);
+
+        TaskGraph tg1 = new TaskGraph("s0") //
+                .transferToDevice(DataTransferMode.FIRST_EXECUTION, a, b) //
+                .task("t0", TestHello::add, a, b, c) //
+                .transferToHost(DataTransferMode.EVERY_EXECUTION, c);
+
+        TaskGraph tg2 = new TaskGraph("s1") //
+                .transferToDevice(DataTransferMode.FIRST_EXECUTION, a, b) //
+                .task("t1", TestHello::add, a, b, c) //
+                .transferToHost(DataTransferMode.EVERY_EXECUTION, c);
```

Comment:
is it possible to add a test in which the graphs have different compute method and they do not share the same variables?